README.md

```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
 
 ![s1](https://raw.githubusercontent.com/Komorebi-Fork/komorebi/master/screenshots/collage.jpg)
 
+## Installing
 
-## How do I install Komorebi?
+Komorebi was tested on:
 
-### Packaged install (easy)
+- **Ubuntu** _18.04_
+- **Elementary OS** _5.1.4_
+- **Pop! OS** _20.04_
+- **Fedora** _32_
+- **Manjaro** _20.0.3_
+- **Deepin** _20_
+- **Arch Linux**
+- **Gentoo**
 
-1. Download `Komorebi` from the [Komorebi releases page](https://github.com/Komorebi-Fork/komorebi/releases).
-2. Install Komorebi using your favorite package manager (aka. double click on it)
-3. Launch Komorebi!
+### Debian and derivated (Ubuntu, Deepin, Elementary OS, Pop! OS, etc...)
```

Comment:
### Debian and derivatives (...) 

I'd remove the reference to "`.deb` package coming soon" - I'd be happy to provide the package I built yesterday at this point and point users at our releases page (especially with a little more polish after work today). It's not Debian repository ready yet, but it should be enough to get us out of trouble for now. 

e.g. "Download the latest .deb package from our releases page and install it with `dpkg -i komorebi_2.2.0-1.deb' (or by double-clicking on the downloaded file."

Later on we can amend the instructions to 'apt install komorebi'

Reply:
Yes, this text was here as a placeholder until the package is available. Would you like me to link to the GDrive link you posted for now, or wait until it's available on Github releases?

Reply:
Let's link to the release page.

Once your PR is ready to go (and if you're happy for me to do so), I'll merge it immediately before tagging the 2.2.0 release & uploading the debian package. :)

Reply:
Alright, I put a link to the releases page :wink: